docs/assets-update.md

```diff
@@ -0,0 +1,67 @@
+# Clear Containers assets updates 
+
+Clear Containers requires additional resources to create a virtual machine
+container. These resources are called
+[Clear Container assets](https://github.com/clearcontainers/runtime/blob/master/docs/architecture/architecture.md#assets)
+which are a `kernel` and a root `filesystem` image. This document describes when
+these components are updated.
+
+## Clear Containers Kernel
+
+The [Clear Containers kernel](https://github.com/clearcontainers/linux) is a
+Linux\* kernel based on the latest vanilla version of the 
+[LTSkernel](https://www.kernel.org/) and the includes
+[patches](https://github.com/clearlinux-pkgs/linux-container/) necessary to run
+Clear Containers. The `LTS` branch is more "stable" and tested that the latest
```

Comment:
Can you replace the word "stable" with something more descriptive?

Reply:
@iphutch  hi, updated
Let me know if this is more clean ( it is already in the  in the PR ) :

 The `LTS` branch is only updated important bugfixes and it is
 more tested that the latest `stable` branch kernel. Using this `LTS` version
 ensures fewer necessary updates.

Reply:
The `LTS` branch undergoes more testing than the latest `stable` branch of the kernel. The `LTS` branch is only updated with important bug fixes and in turn using this branch ensures fewer required updates.